test/ETHStaticAToken.test.ts

```diff
@@ -26,85 +31,113 @@ describe("ETHStaticAToken", function () {
   });
 
   it("should deploy", async () => {
-    const l2tokenFactory = await starknet.getContractFactory("ETHstaticAToken");
+    const L2TokenFactory = await starknet.getContractFactory("ETHstaticAToken");
     const rewAaveContractFactory = await starknet.getContractFactory("rewAAVE");
     const bridgeContractFactory = await starknet.getContractFactory(
       "token_bridge"
     );
+    ProxyFactoryL2 = await starknet.getContractFactory("proxy");
 
-    bridgeContract = await bridgeContractFactory.deploy({
-      governor_address: BigInt(owner.starknetContract.address),
+    proxyTokenBridgeL2 = await ProxyFactoryL2.deploy({
+      proxy_admin: BigInt(owner.starknetContract.address),
+    });
+    proxyL2Token = await ProxyFactoryL2.deploy({
+      proxy_admin: BigInt(owner.starknetContract.address),
     });
 
-    l2token = await l2tokenFactory.deploy({
-      name: 666,
-      symbol: 666,
-      decimals: 4,
-      initial_supply: { high: 0, low: 0 },
+    bridgeContract = await bridgeContractFactory.deploy();
+
+    l2token = await L2TokenFactory.deploy();
+
+    proxiedL2Token = L2TokenFactory.getContractAt(proxyL2Token.address);
+    proxiedTokenBridgeL2 = bridgeContractFactory.getContractAt(
+      proxyTokenBridgeL2.address
+    );
+
+    await owner.invoke(proxyTokenBridgeL2, "initialize_proxy", {
+      implementation_address: BigInt(bridgeContract.address),
+    });
+    await owner.invoke(proxyL2Token, "initialize_proxy", {
+      implementation_address: BigInt(l2token.address),
+    });
+
+    await owner.invoke(proxiedL2Token, "initialize_ETHstaticAToken", {
+      name: 1234n,
+      symbol: 123n,
+      decimals: 18n,
+      initial_supply: { high: 0n, low: 0n },
       recipient: BigInt(user1.starknetContract.address),
       controller: BigInt(owner.starknetContract.address),
     });
 
+    await owner.invoke(proxiedTokenBridgeL2, "initialize_token_bridge", {
+      governor_address: BigInt(owner.starknetContract.address),
+    });
+
     rewAaveTokenL2 = await rewAaveContractFactory.deploy({
       name: 444,
       symbol: 444,
       decimals: 8,
       initial_supply: { high: 0, low: 0 },
       recipient: BigInt(user1.starknetContract.address),
-      owner: BigInt(bridgeContract.address),
+      owner: BigInt(proxyTokenBridgeL2.address),
     });
+
     //set rewAave address on l2 token bridge
-    await owner.invoke(bridgeContract, "set_reward_token", {
+    await owner.invoke(proxiedTokenBridgeL2, "set_reward_token", {
       reward_token: BigInt(rewAaveTokenL2.address),
     });
+    console.log(l2token.address);
```

Comment:
remove `console.log(...)` on the line and the one below

Reply:
ack